src/Fields/Value.php

```diff
@@ -86,7 +87,9 @@ public function antlersValue(Parser $parser, $variables)
         }
 
         if ($this->shouldParseAntlers()) {
-            return $parser->parse($value, $variables);
+            $correctedValue = (new DocumentTransformer())->correct($value);
+
+            return $parser->parse($correctedValue, $variables);
```

Comment:
This is now breaking when using the regex parser. Simply plop a `{{ now format="Y" }}` in a text field with `antlers: true`.

```suggestion
            if (config('statamic.antlers.version') === 'runtime') {
                $value = (new DocumentTransformer())->correct($value);
            }

            return $parser->parse($value, $variables);
            $correctedValue = (new DocumentTransformer())->correct($value);

            return $parser->parse($correctedValue, $variables);
```